packages/components/src/components/ContentCard/ContentCard.test.tsx

```diff
@@ -0,0 +1,111 @@
+import { userEvent } from "@testing-library/user-event";
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { ContentCard } from "./ContentCard";
+
+describe("<ContentCard>", () => {
+  const user = userEvent.setup();
+
+  it("renders text props", () => {
+    render(
+      <ContentCard
+        date="Wed, Dec 27 2023"
+        href=""
+        imageAlt=""
+        imageSrc=""
+        tag="🏠 housing"
+        text="hello! Long text"
+        title="Good title!"
+      />,
+    );
+
+    expect(screen.queryByText("Good title!")).toBeVisible();
```

Comment:
Why using "queryByText"? Why "getByText" is not enough? 🤔

Reply:
oh duh me